PRINCIPLES.md

```diff
@@ -27,31 +27,37 @@ These principles were derived from modern software operations but are rooted in
-
-- These differences could be due to the actual state drifting from the desired state, or the desired state changing intentionally.
-- The source of drift doesn't matter. Contrary to CIops, _any_ drift will trigger a reconciliation
-
-### Principle 4 Notes
+## Glossary
 
-- We talk here about "regular operations." In an emergency, other modes of operations, e.g. manual intervention, should be considered - followed by a reconciliation of the "tainted" system with the declared state. → resolve the conflict between "GitOps principle" and "I need to deal with problems that GitOps doesn't cover"
+- ### Break Glass
 
-## Glossary
+    The temporary suspension of GitOps principles, most often pausing automated _Reconciliation_.
+    While these principles apply to "regular operations," it may sometimes be necessary to pause or sidestep them during an emergency such as incident management.
+    In these cases, other modes of operations should be considered (e.g. manual intervention), followed by any necessary updates to the desired state declarations, and finally resuming reconciliation of the manually changed system with the updated declarations again.
+    Pragmatic exceptions to these guiding principles are expected from time to time during the journey toward a system being fully managed by GitOps.
 
 - ### Continuous
 
-    By "continuous" we adopt the industry standard term to mean reconciliation continues to happen, not that it must be instantaneous.
+    By "continuous" we adopt the industry standard term to mean _Reconciliation_ continues to happen, not that it must be instantaneous.
 
 - ### Declarative Description
 
-    Describing the desired state or behavior of a system without specifying how that state will be achieved, thereby separating between configuration - the desired state - and implementation - the commands, API calls, scripts ... that actually achieve the desired state described in the declarative description.
+    Describing the desired state or behavior of a system without specifying how that state will be achieved, thereby separating between configuration (the desired state) and implementation (the commands, API calls, scripts etc.) that actually achieves the desired state described in the declarative description.
 
 - ### Desired State
 
     The aggregate of all configuration data for a system form its _Desired State_ which is defined as data sufficient to recreate the system so that instances of the system are behaviourally indistinguishable.
 
+- ### Drift
+
+    When a system's _Actual State_ changes for any reason other than its versioned _Desired State_ declarations having changed, we say that the system has drifted from it's _Desired State_.
+
+- ### Reconciliation
+
+    The process of ensuring that the _Actual State_ of a sytem matches it's versioned _Desired State_ declarations.
+    Contrary to CIops, any divergence between the two will trigger reconciliation, regardless of where changes occured.
```

Comment:
should this be "where" or "how" changes occurred?

Reply:
This is meant to convey that unlike event driven CI, reconciliation doesn't only need to be triggered by an intentional change to the desired state stored in Git. Divergence could also happen in the runtime environment, accidentally (by a developer with permissions they shouldn't have), purposefully (by someone authorized to do incident management during an emergency or by a bad actor), or from some system failure, etc. Agents running inside the runtime who watch the desired state store allows divergence from whichever direction to trigger a reconciliation loop.

Maybe this could be worded differently, but this is what "where" was supposed to mean. Any suggestions on clarifying that in a simple way?

Reply:
Actually with that explanation, I see it clearly now. I was probably being pedantic, all good with it as is.